build_tools/cmake/build_e2e_test_artifacts.sh

```diff
@@ -101,7 +101,8 @@ FLAG_DUMP="${E2E_TEST_ARTIFACTS_DIR}/benchmark-flag-dump.txt"
 ./build_tools/benchmarks/export_benchmark_config.py \
   execution \
   --benchmark_presets="${EXECUTION_PRESETS}" \
-  --output="${EXECUTION_CONFIG}"
+  --output="${EXECUTION_CONFIG}" \
+  --shard_count="a2-highgpu-1g=2" # Just an experiment for now
```

Comment:
I'm open to suggestions on how to improve that. `--shard_count` currently accepts a list of device_name-to-shard_count mappings (example: `a2-highgpu-1g=3,c2-standard-16=2`) I don't think that's great.

Of course I could put that into code (basically hard-coded) but I assume we want to be able to overwrite that in a manual workflow run, right?